packages/service-library/src/servicelib/long_running_tasks/task.py

```diff
@@ -162,23 +189,24 @@ async def setup(self) -> None:
         )
         await self._started_event_task_cancelled_tasks_removal.wait()
 
-        # status_update
-        self._task_status_update = create_periodic_task(
-            task=self._status_update,
+        # tasks_monitor
+        self._task_tasks_monitor = create_periodic_task(
+            task=self._tasks_monitor,
             interval=_STATUS_UPDATE_CHECK_INTERNAL,
-            task_name=f"{__name__}.{self._status_update.__name__}",
+            task_name=f"{__name__}.{self._tasks_monitor.__name__}",
         )
-        await self._started_event_task_status_update.wait()
+        await self._started_event_task_tasks_monitor.wait()
 
     async def teardown(self) -> None:
         # ensure all created tasks are cancelled
         for tracked_task in await self._tasks_data.list_tasks_data():
-            await self.remove_task(
-                tracked_task.task_id,
-                tracked_task.task_context,
-                # when closing we do not care about pending errors
-                reraise_errors=False,
-            )
+            # when closing we do not care about pending errors
+            with suppress(TaskNotFoundError):
```

Comment:
really? not even a warning?

Reply:
Yes, if the task is missing we do not care, other errors will be raised. let me adjust the comment